cypress/integration/manufacturing_order_spec.js

```diff
@@ -12,4 +12,45 @@ describe('Manufacturing order Test', function() {
 
     cy.get('.form-field-Barcode').should('exist');
   });
+
+  it('Test if HU editor opens and closes', function() {
+    cy.visit('/window/53009');
+
+    cy.get('.table-flex-wrapper-row')
+      .find('tbody tr').eq(0)
+      .should('exist');
```

Comment:
@siemiatj @metas-ts  IMHO we shall get rid of things like
```
.get('.table-flex-wrapper-row')
.find('tbody tr')
```

Those names are implementation details which a regular test writer shall not be aware of.
More having things like that makes our tests uber-fragile.
wdyt?

Reply:
totally agree

Reply:
It's all true but I don't have the capacity to write custom commands for every operation atm :) It's way faster for me to check console, see what element I need and quickly use this in test.